iaac/terraform/common/nvidia-device-plugin/locals.tf

```diff
@@ -0,0 +1,20 @@
+locals {
+  name = "nvidia-device-plugin"
+  namespace = "nvidia-device-plugin"
```

Comment:
can we install in kube-system?
https://github.com/NVIDIA/k8s-device-plugin/blob/v0.12.3/nvidia-device-plugin.yml#L19

Reply:
done